lib/read_parsers.cc

```diff
@@ -41,6 +41,7 @@ SeqAnParser::SeqAnParser( char const * filename ) : IParser( )
     }
     __asm__ __volatile__ ("" ::: "memory");
     _private->seqan_spin_lock = 0;
+    _num_reads = 0;
```

Comment:
Since `_num_reads` is initialized in the no argument constructor of the superclass there is no need to re-initialize it. 

http://stackoverflow.com/a/120943